pkg/controller/apis/config/v1alpha1/defaults.go

```diff
@@ -22,6 +22,7 @@ import (
 	cmconfigv1alpha1 "k8s.io/controller-manager/config/v1alpha1"
 	kubectrlmgrconfigv1alpha1 "k8s.io/kube-controller-manager/config/v1alpha1"
 	csrsigningconfigv1alpha1 "k8s.io/kubernetes/pkg/controller/certificates/signer/config/v1alpha1"
+	cronjobconfigv1alpha1 "k8s.io/kubernetes/pkg/controller/cronjob/config/v1alpha1"
```

Comment:
Can we maybe split the API part for the config from the controller implementation into two separate PRs?

Reply:
Yeah, that's doable, if needed.